src/state/slices/assetsSlice/assetsSlice.ts

```diff
@@ -138,15 +138,12 @@ export const assetApi = createApi({
           {},
         )
         const data = {
-          byId: assets,
-          ids: Object.keys(assets) ?? [],
+          byId: assetsById,
+          ids: Object.keys(assetsById) ?? [],
         }
-        return { data }
-      },
-      onCacheEntryAdded: async (_args, { dispatch, cacheDataLoaded, getCacheEntry }) => {
```

Comment:
any of these `onCacheEntryAdded` calls are unnecessary relics from when we initially setup RTK and can be removed entirely

Reply:
hasta la vista 3c1777c69d7cf23edbe4416e51226fa408fd3a20